WordPress/src/test/java/org/wordpress/android/ui/posts/AutoSavePostIfNotDraftUseCaseTest.kt

```diff
@@ -160,7 +160,7 @@ class AutoSavePostIfNotDraftUseCaseTest {
     }
 
     private fun createOnPostChangedEvent(post: PostModel, error: PostError? = null): OnPostChanged {
-        val event = OnPostChanged(CauseOfOnPostChanged.RemoteAutoSavePost(post.id), 0)
+        val event = OnPostChanged(CauseOfOnPostChanged.RemoteAutoSavePost(post.id, post.remotePostId), 0)
```

Comment:
@malinajirka I know this is out of the scope of this PR and is occurring due to https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1501/ I added it here so that the tests would pass. I was going to create this fix in another PR and then create a branch from there for this but in the interest of time. Let me know if that's okay. Thanks 🙏

Reply:
Yeah, good call :). I actually took the same approach as you in the `release-notes update` PR. Thanks for adding this clarifying comment ;)!